### PR TITLE
Update to latest logger and log conversion

### PR DESCRIPTION
**Pull request by JoeChapman**
So we can see what's happening when this service is invoked

- update hof-logger and churchill version
- use req.log on controller
- set loglevel to silent on test

### CODE REVIEW

File: controllers/convert.js [hunk 1]
```diff
@@ -14,10 +14,15 @@ module.exports = router.post('/',
     debug('Validated and rendered %s', res.locals.html);
 
     const model = new Model();
+
+    req.log('info', 'html-pdf-converter: Creating PDF');
```
Comment (lennym) [inline controllers/convert.js]: I'd probably make the debug messages be `debug` messages rather than `info`. Otherwise it's just going to be noise.

**Comment by lennym** [comment]
What's the motivation behind prefixing all the logs with `html-pdf-converter`? There's not going to be anything else in the logs for this app, so it just feels a bit like unnecessary noise.

**Comment by JoeChapman** [comment]
Simply to clearly distinguish them from anything else when we are groking the logs

**Comment by lennym** [comment]
Are the logs for multiple services merged into a single stream? I'm just not certain what the "other things" in the logs are?

**Comment by lennym** [comment]
This will help with your test stubs btw: https://github.com/UKHomeOfficeForms/hof-util-reqres/pull/3

**Pull request merged**